eng/common/AzurePipelineTemplates/Maestro-PublishBuildToMaestro-Steps.yml

```diff
@@ -0,0 +1,241 @@
+# Parameters:
+#   AssetNames and AssetVersions can be separated by ';' for publishing build with multiple assets
+#   The lengths when split by ';' must match between both of these parameters
+#     Example:
+#     AssetNames: "PackageName1;PackageName2"
+#     AssetVersions: "1.1;1.2"
+#   
+#   TriggerSubscription will publish the build to the default channel. 
+parameters:

```

Comment:
@kythant , are these folders supposed to be consistent across Maestro-driven repos? Any concerns about the customization here?

Reply:
Talked with Joshua offline to revert these back to the original form and move the modifications outside in a new yml template.